package/package.json

```diff
@@ -56,15 +56,15 @@
   },
   "devDependencies": {
     "@sveltejs/kit": "^1.27.0",
-    "@trpc/client": "^10.41.0",
-    "@trpc/server": "^10.41.0",
+    "@trpc/client": "^11.0.0-next-beta.190",
+    "@trpc/server": "^11.0.0-next-beta.190",
     "@types/ws": "^8.5.8",
     "typescript": "^5.2.2"
   },
   "peerDependencies": {
     "@sveltejs/adapter-node": ">=1.2",
-    "@trpc/client": ">=10",
-    "@trpc/server": ">=10",
```

Comment:
This is too lax - we will definitely break things in the future that this won't warn for